### PR TITLE
Update readme.md

### PR DESCRIPTION
**Pull request by ubani**
added clarification on the documented links

### CODE REVIEW

**Comment by ubani** [comment]
I also added a small tutorial for setting up EasyCLA

**Comment by andorsk** [comment]
This is fantastic @ubani . Thank you for putting this together. We'll put it here, but I think we should also see if this can go to the larger org repo as well.

**Comment by ubani** [comment]
Thank you @andorsk just lmk where and how I can help by pointing a link to me.

File: Contributing.md [hunk 1]
```diff
@@ -1 +1,25 @@
-# Need content
\ No newline at end of file
+# Contribution guide
+
+
+## Pre-requisite of EasyCLA before Submitting Pull Requests (PR)
+
+In order to be able to successfully submit a pull request from your fork you need to sign with EasyCLA from Linux Foundation.  
+The video tutorial can be found [here, assuming you have submitted a PR without prior signing](https://youtu.be/WV9AD9QvSCY?si=RmA4Y-Ttk9weLJlE&t=1428), which would leave you stuck as in this screenshot.
```
Comment (andorsk) [inline Contributing.md:7]: Thanks!

**Review by andorsk — APPROVED** [review]
we should move this to the larger org.

**Pull request merged**